run_configs/jules/build_jules.py

```diff
@@ -5,97 +5,50 @@
 # which you should have received as part of this distribution
 ##############################################################################
 import logging
-import os
 
 from fab.build_config import BuildConfig
-from fab.steps.analyse import Analyse
-from fab.steps.archive_objects import ArchiveObjects
-from fab.steps.cleanup_prebuilds import CleanupPrebuilds
-from fab.steps.compile_fortran import CompileFortran, get_fortran_compiler
-from fab.steps.find_source_files import FindSourceFiles, Exclude
-from fab.steps.grab.fcm import FcmExport
-from fab.steps.grab.prebuild import GrabPreBuild
-from fab.steps.link import LinkExe
-from fab.steps.preprocess import fortran_preprocessor
-from fab.steps.root_inc_files import RootIncFiles
-from fab.util import common_arg_parser
+from fab.steps.analyse import analyse
+from fab.steps.archive_objects import archive_objects
+from fab.steps.cleanup_prebuilds import cleanup_prebuilds
+from fab.steps.compile_fortran import compile_fortran
+from fab.steps.find_source_files import find_source_files, Exclude
+from fab.steps.grab.fcm import fcm_export
+from fab.steps.link import link_exe
+from fab.steps.preprocess import preprocess_fortran
+from fab.steps.root_inc_files import root_inc_files
 
 logger = logging.getLogger('fab')
 
 
-def jules_config(revision=None, two_stage=False, verbose=False):
-
-    # We want a separate project folder for each compiler. Find out which compiler we'll be using.
-    compiler, _ = get_fortran_compiler()
-    config = BuildConfig(
-        project_label=f'jules {revision} {compiler} {int(two_stage)+1}stage',
-        verbose=verbose,
-    )
-
-    logger.info(f'building jules {config.project_label}')
-    logger.info(f"OMPI_FC is {os.environ.get('OMPI_FC') or 'not defined'}")
-
-    two_stage_flag = None
-    # todo: move this to the known compiler flags?
-    if compiler == 'gfortran':
-        if two_stage:
-            two_stage_flag = '-fsyntax-only'
-
-    # A big list of symbols which are used in jules without a use statement.
-    # Fab doesn't automatically identify such dependencies, and so they must be specified here by the user.
-    # Note: there are likely to be differences between revisions here...
-    unreferenced_dependencies = [
-        # this is on a one-line if statement, which fab doesn't currently identify
-        'imogen_update_carb',
-    ]
-
-    config.steps = [
+if __name__ == '__main__':
 
-        FcmExport(src='fcm:jules.xm_tr/src', revision=revision, dst='src'),
-        FcmExport(src='fcm:jules.xm_tr/utils', revision=revision, dst='utils'),
+    revision = 'vn6.3'
 
-        # Copy another pre-build folder into our own.
-        GrabPreBuild(path='/home/h02/bblay/temp_prebuild', allow_fail=True),
```

Comment:
Oh, we should put this step back in. It's a useful test for tolerating a missing prebuilds folder.